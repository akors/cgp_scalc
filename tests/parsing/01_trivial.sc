1 + 1
9-4
4-9
2+1+4+5
123 + -11
99 * 21
