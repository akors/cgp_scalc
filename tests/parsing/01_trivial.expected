2
5
-5
12
112
2079
